src/transformers/models/whisper/processing_whisper.py

```diff
@@ -91,3 +91,9 @@ def decode(self, *args, **kwargs):
         the docstring of this method for more information.
         """
         return self.tokenizer.decode(*args, **kwargs)
+
+    def get_prompt_ids(self, text: str):
+        """Converts prompt text to IDs that can be passed to [`~WhisperForConditionalGeneration.generate`]."""
+        prompt_token_id = self.tokenizer.convert_tokens_to_ids("<|startofprev|>")
+        tokenized_text = self.tokenizer(" " + text.strip(), add_special_tokens=False)["input_ids"] if len(text) else []
+        return [prompt_token_id, *tokenized_text]
```

Comment:
Rather than returning the `prompt_ids` as a list of integers, it would be preferable to have them as a tensor. But even better, `get_prompt_ids()` should use the `return_tensors` argument just like `tokenizer` does, so that the caller can decide between numpy or torch tensors, or a list of integers.